### ISSUE TITLE
Issue with Apple Pay Payment Request Button

### ISSUE THREAD

**Issue by jackcw** [richnologies/ngx-stripe#103]
**Is this a feature request or a bug report?**
Bug
I am using the latest version

**What's gone wrong?**
Using `ngx-stripe-payment-request-button` is producing an error `Cannot read property 'applePay' of null`

**What was meant to happen?**
Display an Apple/Google Pay Button

**Steps to reproduce**
HTML
```
<ngx-stripe-payment-request-button 
[paymentOptions]="paymentRequestOptions"
[elementsOptions]="elementsOptions">
</ngx-stripe-payment-request-button>
```
Typescript
```
elementsOptions: StripeElementsOptions = {
    locale: 'en'
};
paymentRequestOptions: PaymentRequestOptions = {
    country: 'GB',
    currency: 'gbp',
    total: {
      label: `Your order at Test`,
      amount: 100,
    },
    requestPayerName: true,
    requestPayerEmail: true
};
```

**Comment by richnologies** [comment]
Hi @jackcw, sorry for the delay, but test this is always a pain. Thanks for the feedback. 

There was a bug causing that this component only work for Apple Pay and to show the error log you get when the `Payment Request Button` is not available. This has been fixed and a new version `10.1.2` released today.

Not available is usually the case when you are not meeting the prerequisites to use `Payment Request Button`. 

- Add a payment method to your browser.
- Serve your application over HTTPS, both in development and production
- For Apple Pay you will need to verify your domain, both in development and production

I've updated the docs site and create a section specific for this component: https://richnologies.gitbook.io/ngx-stripe/core-concepts/payment-request-button

In that link you will find a complete example that works for me. It shows me the Apple Pay Button. 

Also, you may find useful the Stripe, in particular the section regarding prerequisites: https://stripe.com/docs/stripe-js/elements/payment-request-button#html-js-prerequisites

To be honest I don't have to much experience using this feature, so please let me know if this information helps you.

If you have any more questions, do not hesitate to ask. 

Kind regards,

R

**Comment by jackcw** [comment]
Apologies for the delay. This did fix the problem, thank you so much. Had an issue due to using connect as well which caused further headaches so was investigating to see which it was before saying it fixed.
